src/OpenRiaServices.Client.DomainClients.Http/Framework/Http/BinaryHttpDomainClient.cs

```diff
@@ -502,7 +500,7 @@ internal DataContractSerializerSettings GetSubmitDataContractSettings()
                                 if (typeof(DateTimeOffset) == type || type.IsEnum)
                                     knownTypes.Add(type);
                             }
-                            else if (resolver.TryGetEquivalentContractType(type, out var collectionType))
+                            else if (SubmitDataContractResolver.TryGetEquivalentContractType(type, out var collectionType))
```

Comment:
Why static?

Reply:
I wanted to move the creation of the resolver down